### ISSUE TITLE
Strange divider line

### ISSUE THREAD

**Issue by enricobenedos** [codegrue/card_settings#118]
Good morning,

after I've updated this package version from `1.8.2.` to `1.9.0` I noticed this strange UX issue

<img width="375" alt="Screenshot 2020-06-30 at 22 47 10" src="https://user-images.githubusercontent.com/12233746/86175630-fb255a00-bb23-11ea-82e7-5ff454f43f56.png">

```dart
CardSettings(
        key: _formKey,
        showMaterialonIOS: true,
        children: <CardSettingsSection>[
          CardSettingsSection(
            showMaterialonIOS: true,
            children: <Widget>[
              CardSettingsText(
                showMaterialonIOS: true,
                label: AppLocalizations.of(context).translate('name'),
                icon: const Icon(Icons.person),
                requiredIndicator: requiredIndicator(),
                autovalidate: true,
                validator: (value) {
                  if (value == null || value.isEmpty)
                    return AppLocalizations.of(context)
                        .translate('name_field_validator');
                  return null;
                },
                onChanged: (value) => _name = value,
              ),
              CardSettingsText(
                showMaterialonIOS: true,
                label: AppLocalizations.of(context).translate('surname'),
                icon: const Icon(Icons.person),
                requiredIndicator: requiredIndicator(),
                autovalidate: true,
                validator: (value) {
                  if (value == null || value.isEmpty)
                    return AppLocalizations.of(context)
                        .translate('surname_field_validator');
                  return null;
                },
                onChanged: (value) => _surname = value,
              ),
              CardSettingsEmail(
                showMaterialonIOS: true,
                requiredIndicator: requiredIndicator(),
                icon: const Icon(Icons.mail),
                autovalidate: true,
                validator: (value) {
                  if (value == null || value.isEmpty)
                    return AppLocalizations.of(context)
                        .translate('email_field_validator');
                  else {
                    final bool isEmailOk = GlobalMethods.isEmailValid(value);
                    if (!isEmailOk) {
                      return AppLocalizations.of(context)
                          .translate('email_field_validator2');
                    } else {
                      return null;
                    }
                  }
                },
                onChanged: (value) => _email = value,
              ),
              CardSettingsText(
                showMaterialonIOS: true,
                label: AppLocalizations.of(context).translate('subject'),
                icon: const Icon(Icons.subject),
                requiredIndicator: requiredIndicator(),
                autovalidate: true,
                validator: (value) {
                  if (value == null || value.isEmpty)
                    return AppLocalizations.of(context)
                        .translate('subject_field_validator');
                  return null;
                },
                onChanged: (value) => _subject = value,
              ),
              CardSettingsParagraph(
                showMaterialonIOS: true,
                label: AppLocalizations.of(context).translate('message'),
                icon: Icon(Icons.textsms),
                onChanged: (value) => _message = value,
              ),
            ],
          )
        ],
      ),
```

Any idea about how to fix it?

Thank you

**Comment by codegrue** [comment]
Good eye! This is the line under the last section, not getting clipped by the card border. I added clipping by default in 1.9.9

**Comment by enricobenedos** [comment]
Something has been improved with last package update. But there is also a strange UX effect at the end of the card:

![image](https://user-images.githubusercontent.com/12233746/86510800-488b1b00-bdf3-11ea-99e4-f97eed2ed085.png)

The line was not visible on 1.8.2 and it was more beautiful

**Comment by codegrue** [comment]
extra line fixed in release 1.10.0

**Comment by enricobenedos** [comment]
Now it seems ok, thank you @codegrue

**Comment by enricobenedos** [comment]
Seems that from `1.10.0` to `1.12.0` layout is newly broken
![image](https://user-images.githubusercontent.com/12233746/88714947-d077f280-d11d-11ea-9232-5390046b7b0a.png)

and there is also something like a different shrink wrap or different layout usage approach

**Comment by enricobenedos** [comment]
About the shrink approach I found the problem. On `v1.10.0` the card elevation was fixed at `5.0`, now on `v1.12.0` if it is not specified it is automatically inherited from app theme.
